test/ReactWrapper-spec.js

```diff
@@ -372,7 +372,7 @@ describeWithDOM('mount', () => {
 
     });
 
-    it('should not find components with invalid attributes', () => {
+    it.skip('should not find components with invalid attributes', () => {
```

Comment:
sinon-sandbox will do that by itself - if we're not using that yet (and restoring it afterEach test), we should